draft-ietf-jsonpath-base.md

```diff
@@ -1010,40 +1007,44 @@ The current item is selected if and only if the result is `true`.
 
 ~~~~ abnf
 boolean-expr     = logical-or-expr
-logical-or-expr  = logical-and-expr *("||" logical-and-expr)
+logical-or-expr  = logical-and-expr *(S "||" S logical-and-expr)
                                                       ; disjunction
                                                       ; binds less tightly than conjunction
-logical-and-expr = basic-expr *("&&" basic-expr)      ; conjunction
+logical-and-expr = basic-expr *(S "&&" S basic-expr)  ; conjunction
                                                       ; binds more tightly than disjunction
 
-basic-expr   = exist-expr / paren-expr / (neg-op paren-expr) / relation-expr
+basic-expr   = exist-expr / 
+               [neg-op] paren-expr / 
+               relation-expr
 exist-expr   = [neg-op] path                          ; path existence or non-existence
```

Comment:
definition of `exist-expr` might be not necessary with 

```
basic-expr   = [neg-op] path / 
               [neg-op] paren-expr / 
               relation-expr
```

Reply:
I agree `exist-expr` isn't formally necessary, but it gives us a place to highlight the purpose of the syntax and to give it a meaningful name.

Reply:
that was what I was thinking ... agreed, so let it in.